fastestimator/dataset/data/mnist.py

```diff
@@ -29,7 +32,22 @@ def load_data(image_key: str = "x", label_key: str = "y") -> Tuple[NumpyDataset,
     Returns:
         (train_data, eval_data)
     """
-    (x_train, y_train), (x_eval, y_eval) = tf.keras.datasets.mnist.load_data()
+    home = str(Path.home())
+
+    if root_dir is None:
+        root_dir = os.path.join(home, 'fastestimator_data', 'mnist')
+    else:
+        root_dir = os.path.join(os.path.abspath(root_dir), 'mnist')
+
+    train_data = datasets.MNIST(root_dir, train=True, download=True, transform=None)
+    eval_data = datasets.MNIST(root_dir, train=False, transform=None)
```

Comment:
Do we not need download=True here?

Reply:
yes, I think its a placeholder where it will download if it doesn't exist else will use the already downloaded one